spring-cloud-kubernetes-test-support/src/main/java/org/springframework/cloud/kubernetes/integration/tests/commons/Fabric8Utils.java

```diff
@@ -174,6 +177,42 @@ public static void waitForConfigMapDelete(KubernetesClient client, String namesp
 		});
 	}
 
+	/**
```

Comment:
I've added these two methods in order to be sure that we delete everything, otherwise, with bad timings the integration test I added could fail.